src/languages/es.ts

```diff
@@ -324,6 +324,9 @@ export default {
         action: 'Acción',
         expenses: 'Gastos',
         tax: 'Impuesto',
+        shared: 'Compartido',
+        drafts: 'Borradores',
+        finished: 'Finalizado',
```

Comment:
```suggestion
        finished: 'Finalizados',
```

Reply:
@luacmartins [they got approved](https://swmansion.slack.com/archives/C01GTK53T8Q/p1716884578368599)

Reply:
Haha I asked for another translation and got the plural like I suggested above. I think it makes sense to use the plural, since all other items are plural too, e.g. `Expenses`, `Drafts`

Reply:
Bump @jnowakow. I think we should change this to plural since all other ones are plural too